lib/pages/connect/connect_page_mixin.dart

```diff
@@ -70,11 +71,18 @@ mixin ConnectPageMixin {
       redirectUrl: redirectUrl,
     );
     final urlScheme = _getRedirectUrlScheme(redirectUrl);
+    if (homeserver != null) {
+      html.window.localStorage.addAll({
+        'homeserver': homeserver,
+        'redirectUrl': html.window.location.href,
+      });
```

Comment:
use dart storage instead

Reply:
and ADR to improve later when we have one app connect to dedicated one homeserver